### ISSUE TITLE
how to change line color and fill color?

### ISSUE THREAD

**Issue by romaster93** [wkentaro/labelme#575]
i need to change this line color. because i cant see the mask.

what should i do?

![Screenshot from 2020-01-29 16-43-49](https://user-images.githubusercontent.com/52372619/73336976-a81a5300-42b6-11ea-9201-81fb8690a2a7.png)

**Comment by wkentaro** [comment]
```
labelme --config '{shape_color: null}'
```
Or maybe you can also try
```
labelme --config '{shape_color: auto, shift_auto_shape_color: 1}'
```

**Comment by wkentaro** [comment]
I see this is a bit user-unfriendly. This will be fixed in version 4.1.2.
```
pip install labelme==4.1.2 --no-cache-dir
```
